diff --git a/astropy/utils/iers/tests/test_leap_second.py b/astropy/utils/iers/tests/test_leap_second.py
index a5efebcc0..d446f1efb 100644
--- utils/iers/tests/test_leap_second.py
+++ utils/iers/tests/test_leap_second.py
@@ -12,5 +12,5 @@ from astropy.utils.iers import iers
 from astropy.utils.data import get_pkg_data_filename


-SYSTEM_FILE = '/usr/share/zoneinfo/leap-seconds.list'
+SYSTEM_FILE = '/invalid/path'
 ON_CI = 'CI' in os.environ or 'TRAVIS' in os.environ

 # Test leap_seconds.list in test/data.
